src/structures/MessageSelectMenu.js

```diff
@@ -207,6 +254,14 @@ class MessageSelectMenu extends BaseMessageComponent {
   static normalizeOptions(...options) {
     return options.flat(Infinity).map(option => this.normalizeOption(option));
   }
+  /**
+   * Check if type is a select menu
+   * @param {MessageComponentType} type The select menu type
+   * @returns {boolean}
+   */
+  static checkType(type) {
+    return SelectMenuComponentTypes[type];
```

Comment:
what's the use of this method? also it doesn't returns boolean ig?

Reply:
ah yeah lol